clients/README.md

```diff
@@ -0,0 +1,10 @@
+# Unity Catalog Clients
```

Comment:
we need to generate all the client together in a centralized way. today it is `build/sbt generate`. for python we need to integrate with that.

Reply:
I'm not very familiar with Scala but I tried to add the python client generation to `build.sbt`, and when I run `build/sbt generate`, it seems to properly generate it in `target/clients/python`. 

Since it's now generated in `target/`, I removed the actual generated code from `clients/python` to match the Java client generation behavior. In addition, most of the clutter are removed by disabling tests and documentation generation. However I kept some of the files such as `git_push.sh` and `.github/workflows` since they also seem to be present in the generated Java client. 

Let me know how this looks!